dotCMS/src/integration-test/java/com/dotcms/publisher/bundle/business/BundleFactoryTest.java

```diff
@@ -0,0 +1,215 @@
+package com.dotcms.publisher.bundle.business;
+
+import com.dotcms.datagen.TestDataUtils;
+import com.dotcms.datagen.UserDataGen;
+import com.dotcms.publisher.bundle.bean.Bundle;
+import com.dotcms.publisher.business.DotPublisherException;
+import com.dotcms.publisher.business.PublisherAPI;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.business.FactoryLocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.util.DateUtil;
+import com.dotmarketing.util.UUIDGenerator;
+import com.liferay.portal.model.User;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+
+public class BundleFactoryTest {
+
+    private static BundleFactory bundleFactory;
+    private static User adminUser;
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        //Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+        bundleFactory = FactoryLocator.getBundleFactory();
+        adminUser = APILocator.systemUser();
+    }
+
+    private String insertPublishingBundle(final String userId, final Date publishDate)
+            throws DotDataException {
+        final String uuid = UUIDGenerator.generateUuid();
+        final Bundle bundle = new Bundle();
+        bundle.setId(uuid);
+        bundle.setName("testBundle"+System.currentTimeMillis());
+        bundle.setForcePush(false);
+        bundle.setOwner(userId);
+        bundle.setPublishDate(publishDate);
+        APILocator.getBundleAPI().saveBundle(bundle);
+
+        return uuid;
+    }
+
+    @Test
+    public void test_findSentBundles_byAdminUser() throws DotDataException {
+        //Create a few bundles that the owner is the Admin
+        final ArrayList<String> bundleIdsAdmin = new ArrayList<>();
+        final String systemUserId = adminUser.getUserId();
+        for(int i=0;i<5;i++){
+            bundleIdsAdmin.add(insertPublishingBundle(systemUserId,new Date()));
+        }
+        //Create a few bundles that the owner is a new User
+        final User newUser = new UserDataGen().nextPersisted();
+        final String newUserId = newUser.getUserId();
+        for(int i=0;i<5;i++){
+            bundleIdsAdmin.add(insertPublishingBundle(newUserId, new Date()));
+        }
+
+        List<Bundle> bundlesSent = bundleFactory.findSentBundles(100,0);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'bundlesSent' could be declared final](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4502981)